data_steward/deid/config/ids/tables/visit_occurrence.json

```diff
@@ -1,12 +1,24 @@
         }
```

Comment:
not finding alias `visit_occurrence` in `visit_occurrence.visit_occurrence_id = record2.visit_occurrence_id`